### ISSUE TITLE
Tooltip content isn't announced by the screen reader

### ISSUE THREAD

**Issue by AbdelrhmanMagdy** [OfficeDev/office-ui-fabric-react#12236]
### Environment Information

- **Package version(s)**: 7.76
- **Browser and OS versions**: Windows 10 / Edge Chromium

### Describe the issue:

When i hover over a tooltip button, the screen reader doesn't read the displayed message.

### Please provide a reproduction of the issue in a codepen:
[Codepen](https://codepen.io/abdelrhmanmagdy/pen/wvaPREa)
1- open the codepen in edge browser
2- open windows narrator 
3- move to the button and observe the issue

#### Actual behavior:

The displayed message doesn't get read by the screen reader

#### Expected behavior:

The screen reader should announce the message in the tooltip once the tooltip displayed
